content/manuals/desktop/release-notes.md

```diff
@@ -25,6 +25,44 @@ Take a look at the [Docker Public Roadmap](https://github.com/orgs/docker/projec
 
 ## 4.36.0
 
+{{< release-date date="2024-12-12" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.37.0" build_path="/TBD/" >}}
+
+### New
+
+- TBD
+
+### Upgrades
+
+- [Docker Buildx v0.19.1](https://github.com/docker/buildx/releases/tag/v0.19.1)
+- [Docker Compose v2.31.0](https://github.com/docker/compose/releases/tag/v2.31.0)
+- [Docker Scout CLI v1.15.1](https://github.com/docker/scout-cli/releases/tag/v1.15.1)
+- [NVIDIA Container Toolkit v1.17.2](https://github.com/NVIDIA/nvidia-container-toolkit/releases/tag/v1.17.2)
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- The default disk usage limit for Docker Engine in new installations is now 1TB.
+- Fixed an issue where Docker Desktop would fail to start. Fixes [docker/for-win#14453](https://github.com/docker/for-win/issues/14453)
+- Fixed an issue where containers could not establish loopback `AF_VSOCK` connections.
+- Improved logging when checking filesystems.
```

Comment:
maybe we should not include this one? It seems it impacts mostly us and not customers.

Reply:
all right! removed! 😄